kite_test.go

```diff
@@ -139,6 +145,108 @@ func TestConcurrency(t *testing.T) {
 	wg.Wait()
 }
 
+func TestNoConcurrentCallbacks(t *testing.T) {
+	const timeout = 5 * time.Second
```

Comment:
your max rand sleep time( 99 )ms * 50 (op count) < 5 secs

:)
On Tue, 2 Aug 2016 at 19:23, Rafal Jeczalik <notifications@github.com>
wrote:

> In kite_test.go
> <https://github.com/koding/kite/pull/176#discussion_r73188725>:
>
> > @@ -139,6 +145,108 @@ func TestConcurrency(t *testing.T) {
> >  	wg.Wait()
> >  }
> >
> > +func TestNoConcurrentCallbacks(t *testing.T) {
> > +	const timeout = 5 * time.Second
>
> nah, it's just random without any special meening, 10s seemed too high
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/koding/kite/pull/176/files/589fcf057a108d509fa72b5723e40428f3c08a2c#r73188725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA1sLNWp3dHvoBpjEculRq36TvFVxezdks5qb28HgaJpZM4JaULD>
> .
>
-- 
Cihangir.
Software Engineer @ koding.com